sycl/test/extensions/macro_esimd_emulator.cpp

```diff
@@ -1,15 +1,17 @@
 // This test checks presence of macros for available extensions.
-// RUN: %clangxx -fsycl %s -o %t.out
-// RUN: %t.out
+// RUN: %clangxx -fsycl -fsyntax-only %s
 // REQUIRES: esimd_emulator_be
-#include <iostream>
+
 #include <sycl/sycl.hpp>
-int main() {
+
 #if SYCL_EXT_INTEL_BACKEND_ESIMD_EMULATOR == 1
-  std::cout << "SYCL_EXT_INTEL_BACKEND_ESIMD_EMULATOR=1" << std::endl;
+constexpr bool macro_deifned = true;
```

Comment:
```suggestion
constexpr bool macro_defined = true;
```

Reply:
Fixed in dc8fee6859b425dc1415059b20baa7741668d09f